interfaces/apparmor/spec.go

```diff
@@ -190,6 +278,25 @@ func (spec *Specification) setScope(securityTags []string) (restore func()) {
 	}
 }
 
+var metadataTagsSupported = apparmor_sandbox.MetadataTagsSupported
+
+// MetadataTagSnippet wraps the given AppArmor rule snippet in the given
+// metadata tags if tagging is supported, and returns the resulting snippet.
+// If tagging is not supported, returns the snippet unchanged, without error.
+func MetadataTagSnippet(snippet string, tags []MetadataTag) (string, error) {
+	if len(tags) == 0 || !metadataTagsSupported() {
+		return snippet, nil
+	}
+	tagStrs := make([]string, len(tags))
+	for i, tag := range tags {
+		tagStrs[i] = tag.String()
+	}
+	if strings.Contains(snippet, "\n") {
+		return fmt.Sprintf("tags=(%s) {\n%s\n}", strings.Join(tagStrs, ","), snippet), nil
+	}
+	return fmt.Sprintf("tags=(%s) { %s }", strings.Join(tagStrs, ","), snippet), nil
```

Comment:
though I'd consider wrapping even one snippets in:
```
tags=(a,b,c) {
   ...
}
```


```suggestion
	var b strings.Builder
	nline := strings.Contains(snippet, "\n")
	// generate:
	// tags=(a,b,c) { <snippet>}
	// or
	// tags=(a,b,c) {
	//    <snippet>
	//    <snippet>
	// }
	b.WriteString("tags=(")
	for i, tag := range tags {
		b.WriteString(tag.String())
		if i > 0 {
			b.WriteString(",")
		}
	}
	b.WriteString(") {")
	if nline {
		b.WriteString("\n")
	}
	// snippet
	b.WriteString(snippet)
	if nline {
		b.WriteString("\n")
	}
	b.WriteString("}\n")
	return b.String()
```